identify/extensions.py

```diff
@@ -211,7 +211,7 @@
     'txsprofile': {'text', 'ini', 'txsprofile'},
     'txt': {'text', 'plain-text'},
     'urdf': {'text', 'xml', 'urdf'},
-    'v': {'text', 'verilog'},
+    'v': {'text', 'verilog', 'v', 'vlang'},
```

Comment:
ah yeah we can't have an extension mapping to two different tags

Reply:
> ah yeah we can't have an extension mapping to two different tags

I saw others like `svg` that are both image and XML and thought that was okay, but I don't have a full understanding of identify, so thanks for calling it out!

```py
'svg': {'text', 'image', 'svg', 'xml'},
```

My other thought was that it is unlikely for a project to be working with _both_ Verilog _and_ V files because that would be a headache, but this must not work for other reasons as well? 🤷🏻‍♂️

Does this mean that there is no way for identify to support V? Do you have ideas of how we could handle this?

Thanks for your help!